README.md

```diff
@@ -25,6 +25,15 @@ This is a shortcut for [`autograder.cli.courses.assignments.submissions.submit`]
 python3 -m autograder.run.submit my_file.py
 ```
 
+#### `autograder.run.submit` when late
+
+This command submits a late assingment. For more information and examples,
+see the [late submission section](#submitting-an-assignment-late) of this document.
```

Comment:
This should just be a paragraph in the above section, not a subsection;
e.g., "To submit an assignment late ... see this section ...".